MovieQuiz/Models/GameRecord.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+struct GameRecord: Codable {
+    let correct: Int
+    let total: Int
+    let date: Date
+}
+
+extension GameRecord: Comparable {
+    static func < (lhs: GameRecord, rhs: GameRecord) -> Bool {
+        return lhs.correct < rhs.correct
```

Comment:
Можно еще и короткий вариант использовать
```suggestion
       lhs.correct < rhs.correct
```